### DISCUSSION TITLE
Have to disable liveReload to make hot works?

### DISCUSSION THREAD

**Discussion by allaire** [webpack/webpack#15156] [General]
Hi,

I recently upgraded to Webpack 5 and what I found is that if I don't set `liveReload` to `false`, it reloads the page before `hot` is able to inject the updated code.

In Webpack 4, that wasn't an issue. I'm running Webpack-dev-server in a docker container, here's my config:

```
    devServer: {
      host: "0.0.0.0", // https://github.com/webpack/webpack-dev-server/issues/547#issuecomment-284737321
      port: 3000,
      allowedHosts: "all",
      hot: true,
      liveReload: false,
      proxy: {
        "*": "http://app:3300",
      },
    },
```

Is it normal to have `liveReload: false` so `hot: true` works?

Thanks!

**Comment by alexander-akait** [comment]
Do you have hot code? Can you provide example of the problem?

**Reply by allaire** [reply]
Found the issue, it was caused by multiple entries.

`runtimeChunk: "single", // https://github.com/webpack/webpack-dev-server/issues/2792` fixed it.
